### ISSUE TITLE
Yabai + Tmux + Neovim

### ISSUE THREAD

**Issue by nikitathomas342** [koekeishiya/yabai#2238]
Is there any way to for yabai + tmux + neovim to communicate with each other when doing pane switching. Currently i can get tmux and neovim to work together with a plugin from tmux. But i can't seem to find a way for it to work 3 ways on one hotkey.
For example:
- Left: CTRL+h
- Right: CTRL+l
- Up: CTRL+k
- Down: CTRL+j

**Comment by PhrantiK** [comment]
Yep, I managed to get it working with tmux/nvim/yabai pane nav & resizing across windows & displays:

https://github.com/mrjones2014/smart-splits.nvim/discussions/150

I made a change to the above by adding `|| true` to the end of the yabai commands so it fails quietly otherwise tmux flashes an annoying full-pane message if you accidentally try to navigate with nowhere to go. ie:

`run 'yabai -m window --focus west || yabai -m display --focus west || true'`

The only thing that doesn't work is nested tmux sessions in ssh but I'm saving that problem for the next time I fancy debossed keycap outlines on my forehead trying to figure out tmux config syntax.
